### ISSUE TITLE
Unexpected behaviour when source use tabulations

### ISSUE THREAD

**Issue by Gurimarukin** [philss/floki#210]
Hi!

I came across a strange case where selectors weren't working as expected:

```elixir
def scrap do
  body = """
    <div class="a		">a</div>
  """ # the white spaces above are tabs

  body |> Floki.find(".a")
end
# => []
```

If we escape the tabs:

```elixir
def scrap do
  body = """
    <div class="a		">a</div>
  """

  Regex.replace(~r/\s+/, body, " ") |> Floki.find(".a")
end
# => [{"div", [{"class", "a "}], ["a"]}]
```

I confess, this is a case where the class attribute isn't well formatted. But it's the case in the page I'm scrapping from, so I thought it would be interesting to share it here.

**Comment by philss** [comment]
@Gurimarukin thank you for open the issue, and sorry for the delay.

I think it's something related with how we split the things here: https://github.com/philss/floki/blob/master/lib/floki/selector/attribute_selector.ex#L41

**Comment by fcapovilla** [comment]
I'll try to fix this issue. :)

**Comment by philss** [comment]
@fcapovilla Thank you for the PR! :purple_heart: 

I'm going to test it this week. It looks like it fixes the problem :)

**Comment by Gurimarukin** [comment]
It seems that in the meantime, `HTTPoison` changed something, because I can't reproduce the bug on the body returned from it. (The scraped page still have the weird tabs.)  
Anyway, thanks for the answer!

**Comment by philss** [comment]
Closed by #226. Thank you!
